### PR TITLE
Create ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-4050](https://cumulocity.atlassian.net/browse/DM-4050)
Original PR: [7581](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7581)

[DM-4050]: https://cumulocity.atlassian.net/browse/DM-4050?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: [UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/y2024] (#7581)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]: The Claude AI suggested to use the following title:
```suggestion
title: Firmware update process fails if device reports unexpected UPDATE_RESULT
```

File: content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4050
+version: 1018.503.132
+---
+[UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/y2024] (#7581)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]: The Claude AI suggested to use the following body:
```suggestion
In some cases, devices reported an unexpected UPDATE_RESULT during the firmware update process, but the update was still considered successful. With this change, the firmware update process now fails if a device reports an unexpected UPDATE_RESULT. This prevents incorrectly reporting a successful firmware update when the device actually encountered an error. Devices which have already reported an unexpected UPDATE_RESULT during a firmware update are not affected, but new firmware updates will now properly fail if this situation occurs.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@JacekGonera Please check this PR

**Review by JacekGonera — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4050
+version: 1018.503.132
+---
+Before the change if a device reported an unexpected status or result during a firmware update,
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]:
```suggestion
Previously, if a device reported an unexpected status or result during a firmware update,
```

File: content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+the process would simply wait for a valid state. Now, you can configure LWM2M devices to fail the firmware update
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1018-503-132-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]:
```suggestion
the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**